biz.aQute.bndlib/src/aQute/bnd/osgi/repository/BaseRepository.java

```diff
@@ -25,16 +25,18 @@
 
 import aQute.bnd.exceptions.Exceptions;
 import aQute.bnd.osgi.resource.ResourceUtils;
+import aQute.bnd.service.Tagged;
 
 /**
  * WARNING ! Not tested
  */
-public abstract class BaseRepository implements Repository {
+public abstract class BaseRepository implements Repository, Tagged {
```

Comment:
Maybe RepositoryPlugin should implement Tagged, with a default implementation of no tags?

Reply:
Thanks Peter. Good Points. I'll give it another try when I'm back.